FlorinPosea/teme/3.5.react-rgb-color-game/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
Ai index.html si in folderul public si inafara lui. Nu stiu de ce exact, dar asta se poate sterge. 
favicon.ico, index.html, logo (imaginile png), manifest si robots.txt. Te rog sa le stergi astea care ar trebuii sa fie doar in folderul "public".